src/services/codefixes/importFixes.ts

```diff
@@ -784,6 +787,24 @@ namespace ts.codefix {
                 const symbolId = getUniqueSymbolId(exportSymbolWithIdenticalName, checker);
                 symbolIdActionMap.addActions(symbolId, getCodeActionForImport(moduleSymbol, { ...context, kind: ImportKind.Named }));
             }
+
+            function getEscapedNameForExportDefault(symbol: Symbol): __String | undefined {
+                const declarations = symbol.declarations;
+                if (length(declarations) > 0) {

```

Comment:
Might was well `return firstDefined(symbol.declarations, declaration => ...`

Reply:
That's not equivalent, is it?  It'll check subsequent declarations if the lambda returns `undefined` for the first one?

Reply:
You're right, it's not exactly equivalent, but I don't think it hurts to loop through declarations (and there is usually only one).

Reply:
And the advantage is removing one bounds check?

Reply:
If you like it this way that's fine, I just find it suspicious when we depend on `declarations[0]` being something particular, although that should be work in this case since the symbol for an export assignment or export specifier should always have exactly one declaration.

Reply:
I can check that the length is one if that would be more robust.

Reply:
I don't think that's necessary, it shouldn't really matter whether it has other declarations or not.